bin/download.tk1-exec-uboot

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Since you have the following in your board config file:
download_impl=tegra-uboot-flasher
... this file isn't used any more. Can we remove it?

Reply:
OK done